### PR TITLE
[medium] New Episode Tracker

### PR DESCRIPTION
**Pull request by pikulo-kama**
This widget could be used to get information about on-going series, and watch when new episode would be available.

Public API is being used in script (no need for key)

To find series you would like to follow, you need to search API endpoint by visiting: https://episodate.com/api/show-details?q=<series_code>.

To update series displayed in widget, replace existing series codes in **series** array with yours.

**NOTE:** You could also play around with colors by changing values in **colors** array.

### CODE REVIEW

**Comment by everettjf** [comment]
Great work, thanks

File: widgets/New Episode Tracker/main.jsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+         frame="40,40"
+       />
+    </zstack>
+  )
+}
+
+const Entry = ({info}) => {
+
+  const getNextEpisode = countdown => 
+    s${countdown.season}e${countdown.episode}
```
Comment (everettjf) [inline widgets/New Episode Tracker/main.jsx:81]: something error here
what is s${}e${} syntax ?

Comment (pikulo-kama) [reply widgets/New Episode Tracker/main.jsx:81]: This should be wrapped in quotes, but it's not for some reason

Comment (everettjf) [reply widgets/New Episode Tracker/main.jsx:81]: Got. I have fix it.
Thank you.

**Pull request merged**